clang/lib/AST/Interp/Context.cpp

```diff
@@ -117,7 +118,8 @@ std::optional<PrimType> Context::classify(QualType T) const {
     case 8:
       return PT_Uint8;
     default:
-      return std::nullopt;
+      return PT_IntAP;
+      // return std::nullopt;
```

Comment:
Can this comment be remove?

Reply:
Yes. There are a few such comments left in the patch I think.